python/shotgun_desktop/startup.py

```diff
@@ -417,81 +415,54 @@ def __bootstrap_toolkit_into_classic_config(app, splash, user, pc):
         splash, app, progress_value, message
     )
     mgr.pipeline_configuration = pc["id"]
-    return mgr.bootstrap_toolkit(None)
 
+    def pre_engine_start_callback(tk, ctx):
+        # If the pipeline configuration found in Shotgun doesn't match what we have locally, we have a
```

Comment:
should this be a proper method and not a method inside `__start_engine_in_toolkit_classic`?